pkg/cmd/server/server.go

```diff
@@ -144,6 +147,11 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringSliceVar(&config.restoreResourcePriorities, "restore-resource-priorities", config.restoreResourcePriorities, "desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources")
 	command.Flags().StringVar(&config.defaultBackupLocation, "default-backup-storage-location", config.defaultBackupLocation, "name of the default backup storage location")
 
+	volumeSnapshotLocations := flag.NewMap()
```

Comment:
This was the "best" way I could think to do this, since the default value here is really just `nil` or empty.